src/site/content/en/blog/carousel-best-practices/index.md

```diff
@@ -0,0 +1,164 @@
+date: 2021-01-26
+hero: hero.jpeg
+tags:
+  - blog
+  - fast
+  - performance
+  - web-vitals
+  - internationalization
+---
+
+A carousel is a UX component that displays content in slideshow-like manner. Carousels can "autoplay" or be navigated manually by users. Although carousels can be used elsewhere, they are most frequently used to display images, products, and promotions on homepages.
+
+## Performance
+
+This section discusses the performance impact of carousels and best practices for building performant carousels.
+
+
+### Impact
+
+A well-implemented carousel, in and of itself, should have very minimal or no impact on performance. However, carousels often contain large media assets. Large assets can impact performance regardless of whether they are displayed in a carousel or elsewhere.
+
+
+*   **LCP (Largest Contentful Paint)**
+
+    Large, above-the-fold carousels often contain the page's LCP element, and therefore can have a significant impact on LCP. In these scenarios, optimizing the carousel may significantly improve LCP.
+
+    For an in-depth explanation of how LCP measurement works on pages containing carousels, refer to the [LCP measurement for carousels](#lcp-measurement-for-carousels) section.
+
+*   **FID (First Input Delay)**
+
+    Carousels have minimal JavaScript requirements and therefore should not impact page interactivity. If you discover that your site's carousel has long-running scripts, you should consider replacing your carousel tooling.
+
+*   **CLS (Cumulative Layout Shift)**
+
+    A surprising number of carousels use janky, non-composited animations that can contribute to CLS. On pages with autoplaying carousels, this has the potential to cause infinite CLS. This type of CLS typically isn't apparent to the human eye, which makes the issue easy to overlook. To avoid this issue, [avoid using non-composited animations](/non-composited-animations/) in your carousel (for example, during slide transitions).
+
+
+### Best practices
+
+Optimizing the performance of a carousel includes optimizing both its technical architecture and its content.
+
+
+#### Load carousel content using HTML
+
+Carousel content should be loaded via the page's HTML markup so that it is discoverable by the browser early in the page load process. Using JavaScript to initiate the loading of carousel content is probably the single biggest performance mistake to avoid when using carousels. This delays image loading and can negatively impact LCP.
+
+For advanced carousel optimization, consider loading the first slide statically, then progressively enhancing it to include navigation controls and additional content. This technique is most applicable to environments where you have a user's prolonged attention - this gives the additional content time to load. In environments like home pages, where users may only stick around for a second or two, only loading a single image may be similarly effective.
```

Comment:
```suggestion
For advanced carousel optimization, consider loading the first slide statically, then progressively enhancing it to include navigation controls and additional content. This technique is most applicable to environments where you have a user's prolonged attention—this gives the additional content time to load. In environments like home pages, where users may only stick around for a second or two, only loading a single image may be similarly effective.
```

Reply:
This pops up in a few more places. I can't use search&replace in GH UI so please replace all instances with em dash without surrounding spaces.